order/views.py

```diff
@@ -1,3 +1,48 @@
-from django.shortcuts import render
+import json
 
-# Create your views here.
+from django.http    import JsonResponse
+from django.views   import View
+
+from .models        import *
+
+class ShippingManagementView(View):
+    @login_decorator
+    def post(self, request):
+        data = json.loads(request.body)
+        try:
+            Shipping(
+                    user_id         = data['user_id'],
+                    name            = data['name'],
+                    recipient       = data['recipient'],
+                    address         = data['address'],
+                    phone_number    = data['phone_number']
+                    ).save()
+            return JsonResponse({'message':'SUCCESS POST'}, status=200)
```

Comment:
success 의미는 200에 포함되어 있으므로 status 코드만 리턴해도 됩니다.
```suggestion
            return HttpResponse(status=200)
```